README.adoc

```diff
@@ -14,22 +12,29 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
+* A common issue when people hang out with friends is deciding on what to eat. As a person spends time to scan through their memory for good eateries, the meal timing will be delayed.
```

Comment:
... issue we face when hanging out with friends is ...
as we spend time to....  our meal is delayed